docs/feature-compatibility.txt

```diff
@@ -0,0 +1,331 @@
+.. _laravel-feature-compat:
+
+=============================
+Laravel Feature Compatibility
+=============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, support
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide describes the Laravel features that are supported by
+the {+odm-long+}. This page discusses Laravel version 11.x feature
+availability in the {+odm-short+}.
+
+The following sections contain tables that describe whether individual
+features are available in {+odm-short+}.
+
+Database Features
+-----------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Eloquent Feature
+     - Availability
+
+   * - Configuration
+     - ✓
+
+   * - Read/Write Connections
+     - Use :manual:`read preference </core/read-preference/>` instead.
+
+   * - Multiple Database Connections
+     - ✓
+
+   * - Listening for Query Events
+     - ✓
+
+   * - Monitoring Cumulative Query Time
+     - ✓
+
+   * - Transactions
+     - ✓ See :ref:`laravel-transactions`.
+
+   * - Command Line Interface (CLI)
+     - Use the :mdb-shell:`MongoDB Shell <>` (``mongosh``).
+
+   * - Database Inspection
+     - *Unsupported*
+
+   * - Database Monitoring
+     - *Unsupported*
+
+.. TODO Listening to query events: yes (to be checked)
+.. TODO Monitoring cumulative query time: yes (to be checked)
+
+Query Features
+--------------
+
+The following Eloquent methods are not supported in {+odm-short+}:
+
+- ``toSql()``
+- ``toRawSql()``
+- ``whereColumn()``
+- ``orWhereColumn()``
+- ``whereFulltext()``
+- ``groupByRaw()``
+- ``orderByRaw()``
+- ``inRandomOrder()``
+- ``union()``
+- ``unionAll()``
+- ``havingRaw()``
+- ``having()``
+- ``havingBetween()``
+- ``orHavingRaw()``
+- ``whereIntegerInRaw()``
+- ``orWhereIntegerInRaw()``
+- ``whereIntegerNotInRaw()``
+- ``orWhereIntegerNotInRaw()``
+
+.. list-table::
+   :header-rows: 1
+
+   * - Eloquent Feature
+     - Availability
+
+   * - Running Queries
+     - ✓
+
+   * - Chunking Results
+     - ✓
+
+   * - Aggregates
+     - ✓
+
+   * - Select Statements
+     - ✓
+
+   * - Raw Expressions
+     - *Unsupported*
+
+   * - Joins
+     - *Unsupported*
+
+   * - Unions
+     - *Unsupported*
+
+   * - `Basic Where Clauses <https://laravel.com/docs/11.x/queries#basic-where-clauses>`__
+     - ✓
+
+   * - `Additional Where Clauses <https://laravel.com/docs/11.x/queries#additional-where-clauses>`__
+     - ✓
+
+   * - Logical Grouping
+     - ✓
+
+   * - `Advanced Where Clauses <https://laravel.com/docs/11.x/queries#advanced-where-clauses>`__
+     - ✓
+
+   * - `Subquery Where Clauses <https://laravel.com/docs/11.x/queries#subquery-where-clauses>`__
+     - *Unsupported*
+
+   * - Ordering
+     - ✓
+
+   * - Random Ordering
+     - *Unsupported*
+
+   * - Grouping
+     - Partially supported, use :ref:`Aggregation Builders <laravel-query-builder>`.
```

Comment:
S: since this just links to the top of the Query Builder page, it could be helpful to add an anchor to the "Aggregation" section and link to that